jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -719,3 +720,26 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_jbpm_cluster(){
+    if [ ${KIE_JBPM_CLUSTER} = 'true' ]; then
+      configure_jbpm_cache
+      log_info "KIE_JBPM_CLUSTER enabled"
```

Comment:
seems you forgot to add the env here
https://github.com/jboss-container-images/jboss-kie-modules/pull/434/files#diff-3e7628a0c0671f1c47f2f4da279754577748d791191318fb85c3d250b1ca182cL36

please prefix it with KIE_SERVER_

Reply:
ok, one test if with check of the env and the other is the function that manipulate the xml